### PR TITLE
Created Electric Furnace wiki page

### PR DESCRIPTION
**Pull request by bryansha**
First time contributing to this wiki and I did my best to follow the same design as the other wiki page

Let me know if anything needs fixing thanks!!!

### CODE REVIEW

File: pages/Electric-Furnace.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+The Electric Furnace is a Machine in the [Electric Machines](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) category. It looks like a regular furnace and acts like regular furnaces except it uses electricity rather than other conventional fuel like wood and coal.
+
+## Obtaining
+
+The Electric Furnace is crafted in the [Enhanced Crafting Table](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Enhanced-Crafting-Table). The recipes vary with the machine levels and can be found in the Slimefun guide. The Electric Furnace comes in three tiers
```
Comment (TheBusyBiscuit) [inline pages/Electric-Furnace.md:5]: Missing dot (`.`) at the end here.

File: pages/Electric-Furnace.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+
```
Comment (TheBusyBiscuit) [inline pages/Electric-Furnace.md:8]: ehm, no need for this many new lines.

File: pages/Electric-Furnace.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+## Usage
+
+The Electric Furnaces has to be connected to an [Electrical Network](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines) with sufficient energy to function. 
+
+All three levels of the Electric Furnace have the same layout
+
+And the slots of an Electric Furnace III is shown below:
```
Comment (TheBusyBiscuit) [inline pages/Electric-Furnace.md:15]: * are

File: pages/Electric-Furnace.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+All three levels of the Electric Furnace have the same layout
+
+And the slots of an Electric Furnace III is shown below:  
+![Electric Furnace III Slots](https://raw.githubusercontent.com/Slimefun/Slimefun-Wiki/master/images/electric-furnace.png)
+
+- The two cyan glass panes on the left are the input slots 
+- The orange slots on the right are the output slots which items will appear after they've been processed 
+
+Once the output slots are filled, the Electric Furnaces stops its operation until the output slots are cleared. Therefore, a [Cargo Management System](https://github.com/Slimefun/Slimefun4/wiki/Cargo-Management) is the most efficient way to control the input and output to make an automated system
```
Comment (TheBusyBiscuit) [inline pages/Electric-Furnace.md:21]: Agtain, missing dot at the end.

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
Looks good, some small mistakes were made though.

Also please use lower-case for file extensions, so `.png`, not `.PNG`

**Comment by bryansha** [comment]
All Fixed. Not too familiar with pull request so don't know if the edits worked....

**Comment by Phoenix-Starlight** [comment]
> 
> 
> All Fixed. Not too familiar with pull request so don't know if the edits worked....

Replies about github and how this works. PRs will automatically update to reflect changes in the repo that is being merged into here. So dont worry, your PR will auto-update in response to commits to your fork of the wiki.

**Comment by svr333** [comment]
#93 also adds this page :/

**Comment by bryansha** [comment]
> #93 also adds this page :/

!!!! wow I didn't notice.... I assumed it wasn't done as the link that leads to electric furnace said it wasn't written yet.

Looks like all those links have to be updated soon

**Comment by svr333** [comment]
@bryansha Yeah sorry about this, we really appreciate you making this page
but #93 is still in progress and still requires to be merged, so it isnt fully merged yet and therefore the links arent updated yet

**Pull request closed without merging**